test/cpp/jit/test_graph_executor.cpp

```diff
@@ -29,5 +32,40 @@ TEST(GraphExecutorTest, Basic_CUDA) {
   ASSERT_TRUE(almostEqual(stack[1].toTensor(), r1));
 }
 
+TEST(GraphExecutorTest, runAsync_executor) {
+  /*
+  TODO: there are some problem with C++ parsing script program involving
```

Comment:
Do we have an issue on GIthub to track this issue? If so, we could add it in the comment.
If not, we could open one for tracking purpose.